### PR TITLE
Disable logging for production release environment

### PR DESCRIPTION
**Pull request by olzraiti**
The release branch can be deprecated after these changes, because there is no need to manually comment out all the logging for production release version.

### CODE REVIEW

File: App/proguard-rules.txt [hunk 1]
```diff
@@ -34,9 +34,24 @@
...
+    public static *** e(...);
+    public static *** wtf(...);
+}
+
+#Building with optimization enabled fails without this.
+-optimizations !code/allocation/variable
 
 # We restrict a few more optimizations for the maps library.
--optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
+#-optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
```
Comment (ffagerho) [inline App/proguard-rules.txt]: If this needs to be removed, it's better to actually remove it (and the associated comment) from the file. The lines are always available in the version control history.

Comment (olzraiti) [inline App/proguard-rules.txt]: I actually thought it was initially commented out. I'll uncomment the line, and in the future I'll inspect git diff more cautiously before commiting. Thanks for reviewing this.

**Comment by olzraiti** [comment]
Retest this please

**Comment by olzraiti** [comment]
Retest this please.

**Comment by olzraiti** [comment]
Retest this please.

**Comment by olzraiti** [comment]
Retest this please.

**Comment by olzraiti** [comment]
Retest this please

File: App/build.gradle [hunk 2]
```diff
@@ -5,6 +5,7 @@ android {
     buildToolsVersion '22.0.1'
     lintOptions {
         abortOnError true
+        disable "MissingTranslation"
```
Comment (ffagerho) [inline App/build.gradle]: Instead of this, we could mark the key as non-translatable. May using donottranslate.xml?

**Pull request merged**